spec/spec_helper.rb

```diff
@@ -18,6 +18,8 @@
 SimpleCov.profiles.define 'integration' do
   add_filter 'spec/'
   add_filter 'cli.rb'
+  add_filter '../lib/oneview-sdk/image-streamer'
```

Comment:
Alright, so just add a `/` to the end of the filter, so we can clearly distinguish it is a folder